src/constructs/core/parameters/s3.ts

```diff
@@ -12,3 +12,15 @@ export class GuDistributionBucketParameter extends GuStringParameter {
     });
   }
 }
+
+export class GuPrivateConfigBucketParameter extends GuStringParameter {
+  public static parameterName = "PrivateConfigBucketName";
+
+  constructor(scope: GuStack, id: string = GuPrivateConfigBucketParameter.parameterName) {
+    super(scope, id, {
+      description: "SSM parameter containing the S3 bucket name holding the app's private configuration",
+      default: "/account/services/private.config.bucket",
```

Comment:
Unfortunately there doesn't seem to be a strong convention for this across the department. Accounts which use a single bucket across all apps can make use of this default; others may need to add a few custom parameters in SSM.

Reply:
Hmm, this parameter type forces one to create a parameter store entry for the location of their bucket. I wonder if we should allow either a param store path or a plain string?

Maybe one for a later PR as we understand the use case a bit more?

Reply:
You're right, this could be a bit of a pain... The advantage of doing it this way is that the parameter value is not private (as far as I know!), so if you do need to change the default, it could at least be set via your `riff-raff.yaml` (which is preferable if you use CD for your infrastructure).

Reply:
Makes sense.

I was thinking we ultimately only need a string, which can come from:
- A parameter store parameter
- A string parameter
- Hard coded

Thinking about it more, making it a parameter store parameter makes sense as it follows the convention of the distribution bucket (I want to rename artifact bucket to distribution bucket!)